Makefile

```diff
@@ -118,11 +115,15 @@ cluster-label-worker-rt:
 	@echo "Adding worker-rt label to worker nodes"
 	hack/label-worker-rt.sh
 
+cluster-wait-for-mcp:
+	@echo "Waiting for MCP to be updated"
+	hack/wait-for-mcp.sh
+
 cluster-clean:
 	@echo "Deleting operator"
 	FULL_REGISTRY_IMAGE=$(FULL_REGISTRY_IMAGE) hack/clean-deploy.sh
 
-functests:
+functests: cluster-label-worker-rt cluster-wait-for-mcp
```

Comment:
In the future, we'll probably want to make a new make target that's the aggregate of labelworker+wait+functests. That doesn't have to happen now necessarily though.

we'd do this in the even that someone wanted to setup the cluster in a custom way and just point the func tests at a pre-setup cluster... not a big deal right now though